### PR TITLE
Updating exception details for IotHubSuspendedException

### PR DESCRIPTION
**Pull request by vinagesh**
## Checklist
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [ ] This pull-request is submitted against the `master` branch.

### CODE REVIEW

File: iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs [hunk 1]
```diff
@@ -8,7 +8,8 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when the IoT Hub has been suspended.
+    /// This exception is thrown when the IoT Hub has been suspended. This is likely due to exceeding your
```
Comment (drwill-ms) [inline iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs:11]: Casing should be "IoT hub" with a lowercase 'h' per style guidelines.

Comment (drwill-ms) [reply iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs:11]: FWIW, I prefer not using 2nd person in errors. Feels too personal. I think it sounds less accusatory to phrase it like "This is likely due to exceeding Azure spending limits. To resolve the error, check the Azure bill and ensure there are enough credits."

**Review by drwill-ms — APPROVED** [review]

File: iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs (hunk 1 above)
Comment (vinagesh) [reply iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs:11]: Any suggestions on wording here? "This error is likely due to exceeding Azure spending limits. This error will be resolved when where is enough credits in the Azure account." ?

Comment (vinagesh) [reply iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs:11]: Cool. I'll update.

**Review by jamdavi — APPROVED** [review]

**Pull request merged**